src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -58,7 +58,7 @@ internal override bool Initialize(ApplicationEngine engine)
             });
             engine.Snapshot.Storages.Add(CreateStorageKey(Prefix_FeePerByte), new StorageItem
             {
-                Value = BitConverter.GetBytes(1000L)
+                Value = BitConverter.GetBytes(100L)
```

Comment:
It should be the same as GasPerByte ?
https://github.com/neo-project/neo/pull/1449/files#diff-5e53e8e632a311af22e38f1ebbb5e3a5R14

Reply:
GasPerByte is used to calculate the cost in storage for systemfee, while Prefix_FeePerByte is for network fee calculation.

Reply:
I know, but maybe the costs should be the same..

Reply:
But they are for the different scenerio(bandwidth and local storage). Also, they are not the same at the current stage.